### PR TITLE
fix(table): fix style regression

### PR DESCRIPTION
**Pull request by jcfranco**
**Related Issue:** #7180 

## Summary

✨🏓🔨✨

### CODE REVIEW

File: packages/calcite-components/src/components/table/table.e2e.ts [hunk 1]
```diff
@@ -2034,7 +2034,7 @@ describe("keyboard navigation", () => {
         `#${rowHead.id}`,
         (el) => el.shadowRoot?.activeElement.shadowRoot?.querySelector("th").classList,
       ),
-    ).toEqual({ "0": CSS.numberCell });
+    ).toEqual({ "0": CSS.numberCell, "1": CSS.selectedCell });
```
Comment (jcfranco) [inline packages/calcite-components/src/components/table/table.e2e.ts:2037]: @macandcheese These tests needed to be updated due to [this change](https://github.com/Esri/calcite-design-system/pull/8594/files#diff-4cc3917081bc0192c2695ce4c6e2f58fb22b00f3b5856e6b9a0102754987248dL232). If this is not intentional, we can update `table-header` accordingly and revert 7d1f290a002a4da1081bc4436521feefcaecda9d.

**Review by alisonailea — APPROVED** [review]

File: packages/calcite-components/src/components/action/action.stories.ts [hunk 2]
```diff
@@ -234,3 +234,24 @@ export const darkModeRTL_TestOnly = (): string =>
...
+    --calcite-action-indicator-color: #45B385;
+    --calcite-action-loader-color: #9AE0FF;
+    "
+    scale="s"
+    indicator
+    active
+    text="click-me"
+    label="hello world"
+    text-enabled
+    icon="configure-popup"
```
Comment (alisonailea) [inline packages/calcite-components/src/components/action/action.stories.ts:254]: need to add a "loading" attribute to fully test the tokens here.

File: packages/calcite-components/src/components/action-menu/action-menu.stories.ts [hunk 3]
```diff
@@ -104,3 +104,51 @@ export const openMaxHeight_TestOnly = (): string => html`
 `;
 
 keyDownOpen_TestOnly.parameters = { chromatic: { delay: 1000 } };
+
+export const theming_TestOnly = (): string => html`
+  <calcite-action-menu
+    open
+    style="
+      --calcite-action-menu-close-background-color-active: #9BDAF2;
```
Comment (alisonailea) [inline packages/calcite-components/src/components/action-menu/action-menu.stories.ts:112]: all of these tokens are not covered by the static VRT test

File: packages/calcite-components/src/components/modal/modal.stories.ts [hunk 4]
```diff
@@ -119,9 +119,28 @@ withTooltips_TestOnly.parameters = {
...
+    --calcite-modal-close-button-background-color: #8AA6A3;
+    --calcite-modal-close-button-icon-color: #10403B;
+    --calcite-modal-content-background-color: #8AA6A3;
+    --calcite-modal-content-space: 1rem;
+    --calcite-modal-corner-radius: 0;
+    --calcite-modal-height: 500px;
+    --calcite-modal-scrim-background-color: #4C5958; 
+    --calcite-modal-shadow: 0 0 0 0.25rem #BFBFBF;
+    --calcite-modal-text-color: #10403B;
+    --calcite-modal-width: 600px;"
```
Comment (alisonailea) [inline packages/calcite-components/src/components/modal/modal.stories.ts:142]: all of these tokens are not covered by the VRT test

File: packages/calcite-components/src/components/modal/modal.e2e.ts [hunk 5]
```diff
@@ -594,49 +564,6 @@ describe("calcite-modal", () => {
...
-      `,
-    });
-    const scrimStyles = await page.evaluate(() => {
-      const scrim = document.querySelector("calcite-modal").shadowRoot.querySelector(".scrim");
-      return window.getComputedStyle(scrim).getPropertyValue("--calcite-scrim-background");
-    });
-    expect(scrimStyles.trim()).toEqual("rgba(0, 0, 0, 0.85)");
-  });
-
-  it("when modal css override set, scrim should adhere to requested color", async () => {
```
Comment (alisonailea) [inline packages/calcite-components/src/components/modal/modal.e2e.ts:618]: There is no replacement for these tests ATM

**Review by alisonailea — CHANGES_REQUESTED** [review]
I'd like to revert the removal of theme tests from e2e. The added VRT tests do not fully cover the token values and it is very easy for a human to miss subtle color changes (especially random colors like we are using) resulting in several accepted faulty VRT tests due to human error. Unit tests are better in this case. Since they represent a data application change not an alteration to our underlaying style assignments.

File: packages/calcite-components/src/components/table/table.e2e.ts (hunk 1 above)
Comment (macandcheese) [reply packages/calcite-components/src/components/table/table.e2e.ts:2037]: There were some incorrect merge resolutions that affected other items in Table as well, caught by Chromatic but auto-accepted... can you remove these test updates and I can work through the incorrect merge results?

**Review by jcfranco — COMMENTED** [review]
Per our conversation, will roll back theme-specific test changes, which will be addressed by https://github.com/Esri/calcite-design-system/pull/8993/.

**Pull request merged**